src/language/libraries.md

```diff
@@ -11,7 +11,7 @@ nextpage:
 ---
 
 The `import` and `library` directives can help you create a
-modular and shareable code base. Libraries not only provide APIs, but
+modular and shareable code base. [Libraries][] not only provide APIs, but
 are a unit of privacy: identifiers that start with an underscore (`_`)
 are visible only inside the library. *Every Dart app is a library*, even
```

Comment:
```suggestion
are visible only inside the library. *Every Dart file is a library*, even
```

Technically its a bit more complicated that this description also, because of part files, but I don't know how to simplify this and the distinction is probably not all that important.

The previous "app" description here suggests that the _entire app_ is one library though imo.

Reply:
I could write "*Every Dart file (plus its parts) is a library*".

But I think since "library" will now link to the glossary entry, that now says 

> A library is a single compilation unit, made up of a single primary file _and any optional number of [parts][]_

That kind of covers the caveat. Especially since it's not worth mentioning front and center like you said, I think this is a reasonable compromise to include the info just for people who already want to learn more, so they're clicking the link.

Reply:
Either way is fine with me, although I probably would lean towards adding the `(plus its parts)` since that is fairly short and more accurate.